integrations/nginx/github.go

```diff
@@ -0,0 +1,87 @@
+package nginx
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/akitasoftware/akita-cli/printer"
+)
+
+// The Github API asset type, with most fields missing
+type GithubAsset struct {
+	Name string `json:"name"`
+	ID   int    `json:"id"`
+}
+
+// The Github API release type, with most fields missing
+type GithubRelease struct {
+	TagName string        `json:"tag_name"`
+	Assets  []GithubAsset `json:"assets"`
+}
+
+// Return a list of all the assets available in the latest release
+func GetLatestReleaseAssets() ([]GithubAsset, error) {
+	response, err := http.Get("https://api.github.com/repos/akitasoftware/akita-nginx-module/releases/latest")
+	if err != nil {
+		printer.Debugf("Error performing GET request: %v", err)
+		return nil, err
+	}
+	defer response.Body.Close()
+
+	printer.Debugf("Status: %q\n", response.Status)
+	if response.StatusCode != 200 {
+		return nil, fmt.Errorf("Response code %d from Github", response.StatusCode)
+	}
+
+	decoder := json.NewDecoder(response.Body)
+	var release GithubRelease
+	err = decoder.Decode(&release)
+	if err != nil {
+		printer.Debugf("JSON decode error: %v\n", err)
+		return nil, err
+	}
+
+	return release.Assets, nil
+}
+
+// Download a specific asset (the prebuilt module) to a temporary file
+func DownloadReleaseAsset(id int, filename string) error {
+	download, err := os.Create(filename)
+	if err != nil {
+		printer.Errorf("Can't create destination file: %v\n", err)
+		return err
+	}
```

Comment:
Forgot to close `download`.
```suggestion
	download, err := os.Create(filename)
	if err != nil {
		printer.Errorf("Can't create destination file: %v\n", err)
		return err
	}
	defer download.Close()
```